botocore/response.py

```diff
@@ -80,6 +80,33 @@ def read(self, amt=None):
             self._verify_content_length()
         return chunk
 
+    def readlines(self, delimiter='\n'):
+        """Generator that streams lines. The default delimiter
+        is the newline character.
+
+        If the delimiter argument is specified, it will
+        override the default.
+        """
+        line_buffer = ''
+        BUFFER_SIZE = 2**14 # 16k
```

Comment:
can we have this as an argument, or as a constant at the module level, so it can overrided if needed ?